### ISSUE TITLE
Thymeleaf: Unknown Template Mode 'HTML5'

### ISSUE THREAD

**Issue by nickmelis** [openanalytics/shinyproxy#494]
When running ShinyProxy, the following WARN appears in the logs

```
2024-05-17T13:27:01.907Z  WARN 1 --- [           main] o.thymeleaf.templatemode.TemplateMode    : [THYMELEAF][main] Unknown Template Mode 'HTML5'. Must be one of: 'HTML', 'XML', 'TEXT', 'JAVASCRIPT', 'CSS', 'RAW'. Using default Template Mode 'HTML'.
```

I'm running ShinyProxy 3.1.0 via:

```
docker run openanalytics/shinyproxy:3.1.0
```

Full startup logs below:

```

  .   ____          _            __ _ _
 /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
 \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
  '  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::                (v3.2.2)

2024-05-17T13:26:55.577Z  INFO 1 --- [           main] faultConfiguringBeanFactoryPostProcessor : No bean named 'errorChannel' has been explicitly defined. Therefore, a default PublishSubscribeChannel will be created.
2024-05-17T13:26:55.603Z  INFO 1 --- [           main] faultConfiguringBeanFactoryPostProcessor : No bean named 'integrationHeaderChannelRegistry' has been explicitly defined. Therefore, a default DefaultHeaderChannelRegistry will be created.
2024-05-17T13:26:57.638Z  WARN 1 --- [           main] io.undertow.websockets.jsr               : UT026010: Buffer pool was not set on WebSocketDeploymentInfo, the default pool will be used
2024-05-17T13:26:57.700Z  INFO 1 --- [           main] io.undertow.servlet                      : Initializing Spring embedded WebApplicationContext
2024-05-17T13:26:57.701Z  INFO 1 --- [           main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 6610 ms
2024-05-17T13:26:57.947Z  INFO 1 --- [           main] e.o.c.service.IdentifierService          : ShinyProxy runtimeId:                   b3e453c7-ea84-4549-9dcb-189f6b6f3811
2024-05-17T13:26:58.104Z  INFO 1 --- [           main] e.o.c.service.IdentifierService          : ShinyProxy instanceID (hash of config): fa8f0913d4309dbe1fe44411fc59f5c6d6937837
2024-05-17T13:26:59.896Z  INFO 1 --- [           main] o.s.boot.web.servlet.RegistrationBean    : Filter registration2 was not registered (disabled)
2024-05-17T13:27:00.375Z  INFO 1 --- [           main] o.s.b.a.w.s.WelcomePageHandlerMapping    : Adding welcome page template: index
2024-05-17T13:27:01.841Z  INFO 1 --- [           main] o.s.s.web.DefaultSecurityFilterChain     : Will secure any request with [org.springframework.security.web.session.DisableEncodeUrlFilter@55c20a91, org.springframework.security.web.session.ForceEagerSessionCreationFilter@4a3516fd, org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter@3ba96967, org.springframework.security.web.context.SecurityContextHolderFilter@5e8698c0, org.springframework.security.web.header.HeaderWriterFilter@60484429, org.springframework.web.filter.CorsFilter@5764d421, org.springframework.security.web.csrf.CsrfFilter@3d667b59, org.springframework.security.web.authentication.logout.LogoutFilter@20d3fff4, org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter@443ac5b8, org.springframework.security.web.authentication.www.BasicAuthenticationFilter@b2e1df3, eu.openanalytics.containerproxy.util.AppRecoveryFilter@1feb586d, eu.openanalytics.containerproxy.security.FixedRequestCacheAwareFilter@1237cade, org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter@46590dd0, org.springframework.security.web.authentication.AnonymousAuthenticationFilter@70d7a49b, org.springframework.security.web.session.SessionManagementFilter@2728add3, org.springframework.security.web.access.ExceptionTranslationFilter@731d0d5c, eu.openanalytics.shinyproxy.AuthenticationRequiredFilter@4509b7, org.springframework.security.web.access.intercept.AuthorizationFilter@7865cc83]
2024-05-17T13:27:01.907Z  WARN 1 --- [           main] o.thymeleaf.templatemode.TemplateMode    : [THYMELEAF][main] Unknown Template Mode 'HTML5'. Must be one of: 'HTML', 'XML', 'TEXT', 'JAVASCRIPT', 'CSS', 'RAW'. Using default Template Mode 'HTML'.
2024-05-17T13:27:02.630Z  INFO 1 --- [           main] o.s.l.c.support.AbstractContextSource    : Property 'userDn' not set - anonymous context will be used for read-write operations
2024-05-17T13:27:03.270Z  INFO 1 --- [           main] o.s.i.endpoint.EventDrivenConsumer       : Adding {logging-channel-adapter:_org.springframework.integration.errorLogger} as a subscriber to the 'errorChannel' channel
2024-05-17T13:27:03.272Z  INFO 1 --- [           main] o.s.i.channel.PublishSubscribeChannel    : Channel 'ContainerProxy.errorChannel' has 1 subscriber(s).
2024-05-17T13:27:03.273Z  INFO 1 --- [           main] o.s.i.endpoint.EventDrivenConsumer       : started bean '_org.springframework.integration.errorLogger'
2024-05-17T13:27:03.282Z  INFO 1 --- [           main] io.undertow                              : starting server: Undertow - 2.3.10.Final
2024-05-17T13:27:03.299Z  INFO 1 --- [           main] org.xnio                                 : XNIO version 3.8.8.Final
2024-05-17T13:27:03.327Z  INFO 1 --- [           main] org.xnio.nio                             : XNIO NIO Implementation Version 3.8.8.Final
2024-05-17T13:27:03.399Z  INFO 1 --- [           main] org.jboss.threads                        : JBoss Threads version 3.5.0.Final
2024-05-17T13:27:03.460Z  INFO 1 --- [           main] o.s.b.w.e.undertow.UndertowWebServer     : Undertow started on port 8080 (http)
2024-05-17T13:27:03.847Z  INFO 1 --- [           main] io.undertow.servlet                      : Initializing Spring embedded WebApplicationContext
2024-05-17T13:27:03.848Z  INFO 1 --- [           main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 368 ms
2024-05-17T13:27:04.041Z  INFO 1 --- [           main] o.s.b.a.e.web.EndpointLinksResolver      : Exposing 2 endpoint(s) beneath base path '/actuator'
2024-05-17T13:27:04.220Z  INFO 1 --- [           main] io.undertow                              : starting server: Undertow - 2.3.10.Final
2024-05-17T13:27:04.232Z  INFO 1 --- [           main] o.s.b.w.e.undertow.UndertowWebServer     : Undertow started on port 9090 (http)
2024-05-17T13:27:04.297Z  INFO 1 --- [           main] e.o.c.service.AppRecoveryService         : Recovery of running apps disabled
2024-05-17T13:27:04.299Z  INFO 1 --- [           main] e.o.c.util.StartupEventListener          : Started ShinyProxy 3.1.0 (ContainerProxy 1.1.0)
```

**Comment by nickmelis** [comment]
https://github.com/thymeleaf/thymeleaf-spring/issues/137#issuecomment-292755181 this may be related.

**Comment by LEDfan** [comment]
Hi, thanks for reporting this, we will fix it in the next release. As far as I know this should not cause any issues.

**Comment by LEDfan** [comment]
We just released ShinyProxy 3.1.1 that fixes this, therefore I'm going to close this issue. Thanks again for reporting this.
